plugins/log_request_plugin/src/smartrest.rs

```diff
@@ -66,6 +66,11 @@ pub fn read_tedge_logs(
                 .map(|dt| !(dt < smartrest_obj.date_from || dt > smartrest_obj.date_to))
                 .unwrap_or(false)
         })
+        .filter(|dir_entry| {
+            let file_name = &dir_entry.path();
+            let file_name = file_name.to_str().unwrap();
+            file_name.contains(&smartrest_obj.log_type)
```

Comment:
Wouldn't `starts_with` be a better alternative than `contains`?

Reply:
oh it would, thank you